decidim-core/spec/cells/decidim/address_cell_spec.rb

```diff
@@ -7,42 +7,32 @@
 
   let(:my_cell) { cell("decidim/address", model) }
   let(:address) { "Carrer del Pare Llaurador, 113" }
-  let(:latitude) { nil }
-  let(:longitude) { nil }
-  let(:location_hints) { nil }
-  let(:model) { create(:dummy_resource, address: address, latitude: latitude, longitude: longitude, location_hints: location_hints) }
+  let(:latitude) { 40.1234 }
```

Comment:
Just realized this is for an old Codegram address, could we change it do actually dummy values? (including the address)